lib/commands/project.ts

```diff
@@ -88,38 +206,90 @@ export class CreateNativeCommand extends ProjectCommandBase {
 }
 $injector.registerCommand("create|native", CreateNativeCommand);
 
-export class InitHybridCommand extends ProjectCommandBase {
+export class InitCommand extends InitProjectCommandBase {
+	constructor($project: Project.IProject,
+		$errors: IErrors,
+		$fs: IFileSystem,
+		$logger: ILogger) {
+		super($project, $errors, $fs, $logger);
+	}
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			if(this.isCordovaProject().wait()) {
+				this.$logger.info("Trying to create hybrid project.");
+				this.createProjectFileFromExistingProject(projectTypes.Cordova).wait();
+			} else if(this.isNativeProject().wait()) {
+				this.$logger.info("Trying to create native project.");
+				this.createProjectFileFromExistingProject(projectTypes.NativeScript).wait();
+			} else if(this.isWebsiteProject().wait()) {
+				this.$logger.info("Trying to create mobile website project.");
+				this.createProjectFileFromExistingProject(projectTypes.MobileWebsite).wait();
+			} else {
+				this.$errors.fail("Cannot determine project type. Please specify if the project is hybrid, native or mobile website.");
```

Comment:
Cannot determine project type. Specify project type and try again.

(I'm guessing that as usual we're printing the init help after this error?)